rclcpp/test/rclcpp/test_timer.cpp

```diff
@@ -111,13 +111,13 @@ TEST_F(TestTimer, test_is_canceled_reset)
 
   // reset shouldn't affect state (not canceled yet)
   timer->reset();
-  EXPECT_LE(timer->time_until_trigger().count(), std::chrono::nanoseconds::max().count());
+  EXPECT_LE(timer->time_until_trigger().count(), std::chrono::milliseconds::max().count());
```

Comment:
Why this change?

Reply:
That was a mistake on my side - I don't recall why this was even needed. Restored